dockerfiles/pyopenms/manylinux/Dockerfile

```diff
@@ -1,38 +1,52 @@
 FROM quay.io/pypa/manylinux2014_x86_64
-
-#defaults, can be overwritten from outside
-ARG OPENMS_BRANCH="master" 
+ARG CONTRIB_BUILD_DIR="/contrib-build"
 ARG OPENMS_VERSION="latest"
 
 # make source files from context available in docker
 COPY . /contrib
 
-RUN yum install -y wget
-RUN yum install -y xz qt5-qtbase-devel 
-RUN yum install -y qt5-qtbase-devel qt5-qtsvg qt5-qtsvg-devel python-devel 
-# RUN yum install -y libsvm-devel glpk-devel libzip-devel zlib-devel xerces-c-devel bzip2-devel sqlite-devel hdf5-devel
-RUN yum install -y libtool
+RUN yum install -y \
+    # build system dependencies
+    wget \
+    xz \
+    libtool \
+    cmake3 \
+    # OpenMS and PyOpenMS build dependencies
+    libsvm-devel \
+    glpk-devel \
+    libzip-devel \
+    zlib-devel \
+    xerces-c-devel \
+    bzip2-devel \
+    hdf5-devel \
+    qt5-qtbase-devel \
+    qt5-qtsvg-devel \
+    eigen3-devel \
+    python-devel
+# TODO: do we need sqlite-devel?
+# TODO: can we install boost from yum?
+# TODO: is coin-or-CoinUtils-devel useful?
+# TODO: do we need KissFFT? It's not installed in the regular contrib Dockerfile
 
-RUN yum install -y cmake3
 RUN ln -s /usr/bin/cmake3 /usr/bin/cmake
 
 # builds contrib for the current checked out branch
 # during cleanup: removes archives
-RUN mkdir contrib-build
-RUN ls -l
-WORKDIR /contrib-build
-RUN ls -l
-RUN cmake -DBUILD_TYPE=ALL ../contrib && rm -rf archives src
-    
+WORKDIR ${CONTRIB_BUILD_DIR}
+RUN cmake -DBUILD_TYPE="BOOST;COINOR;KISSFFT;OPENMP" ../contrib && rm -rf archives src
```

Comment:
this currently doesn't work because our IT is down (again)

Reply:
On Linux you probably don't need contrib.
I guess even centos7 has everything we need.

Reply:
that would simplify things a lot

Reply:
I was too quick. Should have known that CoinOR is basically available nowhere..

Reply:
yes, IIRC this was one of the blockers of using yum for everything